shipyard-controller/debug-ui/fetch_sequences.js

```diff
@@ -0,0 +1,87 @@
+function fetchSequences() {
+  const project = document.getElementById("projectSelector").value;
```

Comment:
I would pass the project as a parameter of the function. This way it is easier to test it and if you change the HTML id you don't need to touch this file.